configs/default.yaml

```diff
@@ -1,80 +1,19 @@
 parser: babel-eslint
 
 extends:
-  - eslint:recommended
   - prettier
-  - prettier/react
+  - ./base
 
 plugins:
   - babel
-  - prettier
-  - react
   - vazco
```

Comment:
`vazco` plugin is in the `base` config, so you should be able to remove it from here